llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -274,6 +274,14 @@ static cl::opt<bool>
                  cl::desc("propagate shadow through ICmpEQ and ICmpNE"),
                  cl::Hidden, cl::init(true));
 
+static cl::opt<uint> ClEmbedFaultingInst(
```

Comment:
should this be an enum with  descriptive names?

Reply:
Done